src/translib/transformer/sw_portchannel.go

```diff
@@ -49,14 +55,74 @@ func validateLagExists(d *db.DB, lagTs *string, lagName *string) error {
     return nil
 }
 
+func get_min_links(d *db.DB, lagName *string, links *uint16) error {
+    intTbl := IntfTypeTblMap[IntfTypePortChannel]
+    curr, err := d.GetEntry(&db.TableSpec{Name:intTbl.cfgDb.portTN}, db.Key{Comp: []string{*lagName}})
+    if err != nil {
```

Comment:
We would hit this case when the user tries to create the lag with min-links configured, right?

Reply:
yes.. It's a new PortChannel with min-links request coming in.